src/app/organizations/manage/groups.component.ts

```diff
@@ -142,10 +165,28 @@ export class GroupsComponent implements OnInit {
         });
     }
 
+    async resetPaging() {
+        this.pagedGroups = [];
+        this.loadMore();
+    }
+
+    isSearching() {
+        return this.searchService.isSearchable(this.searchText);
+    }
+
+    isPaging() {
+        const searching = this.isSearching();
+        if (searching && this.didScroll) {
+            this.resetPaging();
+        }
+        return !searching && this.groups.length > this.pageSize;
+    }
+
     private removeGroup(group: GroupResponse) {
         const index = this.groups.indexOf(group);
         if (index > -1) {
             this.groups.splice(index, 1);
+            this.resetPaging();
```

Comment:
Same comment here, would removing a group reset the user back to the beginning of their paging/scrolling?

Reply:
Same as above!